package.json

```diff
@@ -11,7 +11,7 @@
     "start": "./scripts/clear-cloudflare-cache.js; NODE_ENV=production node server.js",
     "format-staged": "pretty-quick --staged --no-restage --bail",
     "format-check": "prettier --check '{.,pages/**,content/docs/**,src/**}/*.{js,md,json}'",
-    "lint-check": "eslint --ext .json,.js src middleware",
+    "lint-check": "tsc --noEmit && eslint --ext .json,.js,.ts,.tsx src middleware --quiet",
```

Comment:
do we need `--skipLibCheck`? like we had in blog?

Reply:
Yep. Good options, it can increase typechecking performance. Added.